src/dom-utils.ts

```diff
@@ -53,3 +53,7 @@ export function updateZIndex() {
 function isHTMLElement(node: Node): node is HTMLElement {
   return node.nodeType === Node.ELEMENT_NODE;
 }
+
+export function setExternalWebChatLanguage() {
```

Comment:
```suggestion
export function getWebChatLanguageAttributeValue() {
```

Just a naming suggestion, 'setXXX' implies we are updating a value rather than reading it

Reply:
Yes, that's correct. I have effect the change